internal/pkg/scorecard/helpers/versions.go

```diff
@@ -39,6 +39,9 @@ func ValidateVersion(version string) error {
 }
 
 func IsV1alpha2() bool {
+	if viper.Sub("scorecard").GetString(VersionOpt) == "" && LatestScorecardVersion == v1alpha2 {
+		return true
+	}
 	if viper.Sub("scorecard").GetString(VersionOpt) == v1alpha2 {
 		return true
 	}
```

Comment:
Maybe simplify this to:
```go
func IsV1alpha2() bool {
	version := viper.Sub("scorecard").GetString(VersionOpt)
	return version == "" || version = v1alpha2
}
```

Reply:
I ended up pulling the viper dependency out of this file completely, instead just passing in the version from the caller, seems simpler with that.
looks like this now:

func IsV1alpha2(version string) bool {
        if version == v1alpha2 {
                return true
        }
        return false
}